gatekeeperv2.kvp

```diff
@@ -0,0 +1,86 @@
+Meta.DisplayName=GatekeeperV2 Bot
+Meta.Description=AMP Discord Bot created by k8thekat - https://github.com/k8thekat/GatekeeperV2
+Meta.OS=Linux
+Meta.Arch=x86_64
+Meta.Author=k8thekat, IceOfWraith
+Meta.URL=https://github.com/k8thekat/GatekeeperV2
+Meta.DisplayImageSource=url:https://avatars.githubusercontent.com/u/68672235?v=4
```

Comment:
The appearance of this image in the instance tile triggers me.

Reply:
What is triggering about it?

Reply:
Doesn't fit properly and is only partly displayed. Lol

Reply:
And I am a dog person 😂

Reply:
I won't ruin your day by pointing out that none of the images fit from Steam and that AMP zooms in and cuts off all the edges of them. Definitely won't tell you about that. ;)